README.md

```diff
@@ -131,14 +123,10 @@ Valid properties for landmark specification referenced by `landmark_url`:
 
 * RID - (string) The ID of the Landmark.
 * mesh - (string) The ID of the Mesh this landmark points
+* anatomy - (string) Name of the anatomy this landmark points
```

Comment:
add "If given, the value for `anatomy` will be used as the display label for the landmark."